pageserver/src/lib.rs

```diff
@@ -62,6 +63,7 @@ pub async fn shutdown_pageserver(deletion_queue: Option<DeletionQueue>, exit_cod
         Duration::from_secs(1),
     )
     .await;
+    tracing::info!("shutdown: page service listeners stopped.");
```

Comment:
note that the timed already logs completions, so each of these would now print two messages for the same one thing.